validation/src/main/scala/hmda/validation/engine/lar/syntactical/LarSyntacticalEngine.scala

```diff
@@ -0,0 +1,28 @@
+package hmda.validation.engine.lar.syntactical
+
+import hmda.model.fi.lar.LoanApplicationRegister
+import hmda.validation.api.ValidationApi
+import hmda.validation.engine.lar.LarCommonEngine
+import hmda.validation.rules.lar.syntactical.{ S010, S011, S020, S040 }
+
+trait LarSyntacticalEngine extends LarCommonEngine with ValidationApi {
+
+  def checkSyntactical(lar: LoanApplicationRegister): LarValidation = {
+    val checks = List(
+      S010,
+      S020
+    ).map(check(_, lar))
+
+    validateAll(checks, lar)
+  }
+
+  def checkSyntacticalIterable(lars: Iterable[LoanApplicationRegister]): LarsValidation = {
```

Comment:
Could they both be `checkSyntactical` (with different signatures)?  (I'm not sure I like that idea, but I'll still put it out there.)

To me `Collection` still sounds like it refers to implementation. (I blame Java.)

Hm. Most things I can think of that read well (to me) end up either involving a lot of words, or having the Validity and Syntactical engines both be classes that get instantiated, rather than traits that get mixed in. (That is, they end up having names that would be the same for both of those engines.)